### PR TITLE
Html tables

### PR DESCRIPTION
**Pull request by dcatan89**
Screen Shot 2021-11-30 at 4 55 53 PM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  </head>
+  <h1>Coding Form!</h1>
+
+  <form action="">
+    <label for="email"> Email </label>
+    <div>
+      <input type="email" name="email" id="email" required />
+    </div>
+
+    <div>Why is coding cool?</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:14]: This text should be in a `label`

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <select name="Options">
+      <option value="Select an option">Select an option</option>
+      <option value="Because its fun">It is fun</option>
+      <option value="Beause it's powerful">Because it's powerful</option>
+      <option value="Because it's interesting">Becuase it's interesting</option>
+    </select>
+    <div>Favorite Language (choose one)</div>
+    <input type="radio" name="coding language" value="HTML" />
+    HTML
```
Comment (codingcodymiller) [inline html-forms/index.html:23]: This text should be in a `label`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <input type="radio" name="coding language" value="CSS" />
+    CSS
```
Comment (codingcodymiller) [inline html-forms/index.html:25]: This text should be in a `label`

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <input type="radio" name="coding language" value="JavaScript" />
+    JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html:27]: This text should be in a `label`

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+    <div>Would you like to sign up for more information?</div>
+    <input type="checkbox" name="sign me up" value="Sign me up" />
+    Sign Me Up!
```
Comment (codingcodymiller) [inline html-forms/index.html:31]: This text should be in a `label`

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+    <div>Comments</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:33]: This text should be in a `label`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 6 above)
Comment (dcatan89) [reply html-forms/index.html:33]: Committed in wrong branch previously however most recent commit is the correct branch.

File: html-forms/index.html (hunk 5 above)
Comment (dcatan89) [reply html-forms/index.html:31]: Committed in wrong branch previously however most recent commit is the correct branch.

File: html-forms/index.html (hunk 4 above)
Comment (dcatan89) [reply html-forms/index.html:27]: Committed in wrong branch previously however most recent commit is the correct branch.

File: html-forms/index.html (hunk 3 above)
Comment (dcatan89) [reply html-forms/index.html:25]: Committed in wrong branch previously however most recent commit is the correct branch.

File: html-forms/index.html (hunk 2 above)
Comment (dcatan89) [reply html-forms/index.html:23]: Committed in wrong branch previously however most recent commit is the correct branch.

File: html-forms/index.html (hunk 1 above)
Comment (dcatan89) [reply html-forms/index.html:14]: Committed in wrong branch previously however most recent commit is the correct branch.

**Comment by dcatan89** [comment]
Fixed indentation and added proper closing tags for body element

File: html-tables/index.html [hunk 7]
```diff
@@ -6,37 +6,38 @@
...
-      <td>Git</td>
-      <td>90</td>
-    </tr>
-  </tbody>
-  </table>
+      <tr>
+        <th scope="col"><b> Student Name</b></th>
+        <th scope="col"><b> Course</b></th>
+        <th scope="col"><b> Grade</b></th>
+      </tr>
```
Comment (codingcodymiller) [inline html-tables/index.html]: This should be in a `thead`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-tables/index.html (hunk 7 above)
Comment (dcatan89) [reply html-tables/index.html]: changes have been made, thank you

File: html-tables/index.html [hunk 8]
```diff
@@ -6,11 +6,11 @@
   <body>
     <h1>Student Grade Table</h1>
     <table>
-      <tr>
+      <thead>
```
Comment (codingcodymiller) [inline html-tables/index.html:9]: You still need a `tr`, it just needs to be a child of the `thead` element. You should review the documentation for this exercise again, you seem to have an incomplete understanding of the concept.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-tables/index.html (hunk 8 above)
Comment (dcatan89) [reply html-tables/index.html:9]: Went back to review and now understand fully, thank you.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**